packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -1857,6 +1860,8 @@ export class ContainerRuntime
 		// messages once a batch has been fully processed.
 		this.scheduleManager.beforeOpProcessing(message);
 
+		this._processedClientSequenceNumber = message.clientSequenceNumber;
```

Comment:
@vladsud Double checking if this is the correct way to track changes in clientSequenceNumber mid batch

Reply:
I think so, but please see my comment below.